docs/introduction/shaders.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I think this part is too thin for a separate section. I think it's OK to be contained in the material section for now. Perhaps as the API and the ecosystem around shaders possibly matures, we move it out a separate section?

Reply:
The non-duplicated part here that seems worth keeping is the reference to shader repositories like ShaderToy and/or ShaderFrog (there are probably others too) and pointers toward components that help with reusing those.  Where would you suggest that appear in `material` documentation?

Reply:
Perhaps when shaders are first introduced.

Reply:
Looks like your shaderfrog gallery went away, was that intentional?  Changed reference to just the shaderfrog site.

Reply:
Yeah, it was an empty repo 🙄